react/FileSelector/Readme.md

```diff
@@ -0,0 +1,48 @@
+This component is used to display a customizable file selector.
+
+### Default
+
+```jsx
+<FileSelector
```

Comment:
I'm not a big fan of `FileSelector` name. Since UI is tied to Cozy, I'm expecting that `FileSelector` let me select a file in Drive :/.

Reply:
Maybe `InputFile` is just fine in fact?

Reply:
Or `FileInput`?